cli/src/tasks/build.ts

```diff
@@ -14,6 +14,9 @@ export interface BuildCommandOptions {
   androidreleasetype?: 'AAB' | 'APK';
   signingtype?: 'apksigner' | 'jarsigner';
   configuration: string;
+  xcodeSigningType: 'automatic' | 'manual';
```

Comment:
```suggestion
  xcodeSigningType?: 'automatic' | 'manual';
```
should be marked as optional